### PR TITLE
Added support keyboard CC67

### PR DESCRIPTION
**Pull request by HorrorTroll**
## Description

Added support keyboard CC67

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/xcstore/cc67/board.h [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/* Copyright 2022 XCStore
```
Comment (zvecr) [inline keyboards/xcstore/cc67/board.h:1]: This should be removed

File: keyboards/xcstore/cc67/matrix.c [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+__attribute__((weak)) void matrix_init_user(void) {}
+
+__attribute__((weak)) void matrix_scan_user(void) {}
+
+// helper functions
+void matrix_init(void)
+{
+    //debug_enable = true;
+    palSetLineMode(LINE_PB12,         PAL_MODE_OUTPUT_PUSHPULL);
```
Comment (zvecr) [inline keyboards/xcstore/cc67/matrix.c:57]: This should be refactored to 
* use `lite`
* use QMK standard GPIO defines

File: keyboards/xcstore/cc67/rules.mk [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+
+# Enter lower-power sleep mode when on the ChibiOS idle thread
+OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
+
+# Custom matrix
+CUSTOM_MATRIX = yes
```
Comment (zvecr) [inline keyboards/xcstore/cc67/rules.mk:27]:
```suggestion
CUSTOM_MATRIX = lite
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/xcstore/cc67/matrix.c (hunk 2 above)
Comment (HorrorTroll) [reply keyboards/xcstore/cc67/matrix.c]: done, just refactor to use lite instead full

**Review by drashna — APPROVED** [review]

File: keyboards/xcstore/cc67/config.h [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* Key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 15
+
```
Comment (fauxpark) [inline keyboards/xcstore/cc67/config.h:25]:
```suggestion
```

File: keyboards/xcstore/cc67/config.h [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+#define BOOTMAGIC_LITE_COLUMN 0
+
+/* Forcing to use NKRO instead 6KRO */
+#define FORCE_NKRO
+
+#ifdef RGB_MATRIX_ENABLE
+    #define RGBLED_NUM 81
+    #define DRIVER_LED_TOTAL RGBLED_NUM
+    #define RGB_MATRIX_MAXIMUM_BRIGHTNESS 200
+    #define RGB_MATRIX_STARTUP_VAL RGB_MATRIX_MAXIMUM_BRIGHTNESS
```
Comment (fauxpark) [inline keyboards/xcstore/cc67/config.h:47]:
```suggestion
    #define RGB_MATRIX_DEFAULT_VAL 255
```

File: keyboards/xcstore/cc67/config.h [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+/* Bootmagic reset */
+#define BOOTMAGIC_LITE_ROW 0
+#define BOOTMAGIC_LITE_COLUMN 0
+
+/* Forcing to use NKRO instead 6KRO */
+#define FORCE_NKRO
+
+#ifdef RGB_MATRIX_ENABLE
+    #define RGBLED_NUM 81
+    #define DRIVER_LED_TOTAL RGBLED_NUM
```
Comment (fauxpark) [inline keyboards/xcstore/cc67/config.h:45]:
```suggestion
    #define RGB_MATRIX_LED_COUNT 81
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by HorrorTroll** [comment]
can this PR re-open? I've just pushed new update commit

**Comment by fauxpark** [comment]
Can't be reopened.

**Pull request closed without merging**